### DISCUSSION TITLE
Problem 4: Improved 3D Planar Measurement Tool

### DISCUSSION THREAD

**Discussion by ckocauw** [chulminy/CIVE497-CIVE700#167] [Q&A]
Hi Prof.,

In this question, I find a little bit different measurements from your solution, which can be seen in your demonstration video. Is this a problem?

Best regards,
Cem

**Comment by chulminy** [comment]
As mentioned in the lecture, since you didn't fix the lens distortion, you can't make the correct measurement.  Don't worry. However, the error should be less than 1cm. 

Chul Min

**Reply by ckocauw** [reply]
Thank you Prof.
